packages/paste-core/primitives/combobox/stories/mutliselect.stories.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from 'react';
+import difference from 'lodash/difference';
+import {useUIDSeed} from '@twilio-paste/uid-library';
+import {Box} from '@twilio-paste/box';
+import {Input} from '@twilio-paste/input';
+import {Label} from '@twilio-paste/label';
+import {FormPill, FormPillGroup, useFormPillState} from '@twilio-paste/form-pill-group';
+import {ComboboxListbox, ComboboxListboxGroup, ComboboxListboxOption} from '@twilio-paste/combobox';
+
+import type {StoryFn} from '@storybook/react/dist/ts3.9/client/preview/types-6-0';
+
+import {ComboboxPrimitive, useComboboxPrimitive, useMultiSelectPrimitive} from '../src';
+
+const items = ['Alert', 'Anchor', 'Button', 'Card', 'Heading', 'List', 'Modal', 'Paragraph'];
+
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: 'Primitives/Combobox',
+  component: ComboboxPrimitive,
+};
+
+const {FunctionAddSelectedItem, FunctionRemoveSelectedItem} = useMultiSelectPrimitive.stateChangeTypes;
+
+export const BasicMultiCombobox: StoryFn = () => {
+  const seed = useUIDSeed();
+  const [filteredItems, setFilteredItems] = React.useState([...items].sort());
+
+  const formPillState = useFormPillState();
+
+  const {
+    getSelectedItemProps,
+    getDropdownProps,
+    addSelectedItem,
+    removeSelectedItem,
+    selectedItems,
+  } = useMultiSelectPrimitive({
+    onSelectedItemsChange: ({type, selectedItems: currentSelectedItems}) => {
+      if (type === FunctionAddSelectedItem) {
+        const [toRemove] = difference(currentSelectedItems, selectedItems);
+        setFilteredItems((state) => state.filter((item) => item !== toRemove));
```

Comment:
If `toRemove` is an array of multiple items, wouldn't `item` never equal `toRemove`?

Reply:
Admittedly, this is a bit brittle, but since `addSelectedItem` and `removeSelectedItem` only handle a single item, I made the assumption that the diff count would always be one.

Since this is an example in our docs, I think it's Good Enough™️. It would be a bug in the hook if this were not triggered per change.